book/src/leader-rotation.md

```diff
@@ -50,3 +138,22 @@ previous leader's slot. Note that the next leader should do repair requests in
 parallel, and postpone sending ticks until it is confident other validators
 also failed to observe the previous leader's entries. If a leader incorrectly
 builds on its own ticks, the leader following it must replace all its ticks.
+
+## Alternative Leader Scheduler Rotation
+
```

Comment:
Could you explain the deficiency in the first proposal here? Also, ideally, you could describe this as a modification of the first, which I think you can. I think the only difference here is that the leader stuffs a transaction on the ledger to signal each validator to generate a leader schedule.  Simpler code vs security concerns.  Seems like the security concern could only be alleviated by implementing all the same logic as in the first proposal. If so, delete this whole section to avoid confusion.

Reply:
@garious I'll delete it.  The problem with this approach is that until the block turning on the new leader schedule is finalized, it is also potentially in flux since the cluster could have an epoch long partition between nodes that accepted that block and those that didn't.